### PR TITLE
rootfs: Add rootfs based on EulerOS 2

### PR DESCRIPTION
**Pull request by liangchenye**
Signed-off-by: Liang Chenye <liangchenye@huawei.com>

### CODE REVIEW

**Comment by liangchenye** [comment]
1. Using official [EulerOS repo](http://developer.huawei.com/ict/site-euleros/euleros/repo/yum/2.2/os/x86_64/)
2. Using [EulerOS 2.2 official docker image](https://hub.docker.com/_/euleros/)
3. using 'yum' only to install packages
PTAL

**Comment by liangchenye** [comment]
travis does not work yet #13

File: rootfs-builder/euleros/rootfs_lib.sh [hunk 1]
```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+#
+# Copyright (c) 2017 Intel Corporation
```
Comment (jodh-intel) [inline rootfs-builder/euleros/rootfs_lib.sh]: This copyright should probably be Huawei, or Intel+Huawei?

/cc @grahamwhaley

File: rootfs-builder/euleros/rootfs_lib.sh [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+debuglevel=2
+logfile=/var/log/yum-euleros.log
+exactarch=1
+
+[Base]
+name=EulerOS-2.2 Base
+baseurl=http://developer.huawei.com/ict/site-euleros/euleros/repo/yum/2.2/os/x86_64/
+enabled=1
+gpgcheck=1
+gpgkey=http://developer.huawei.com/ict/site-euleros/euleros/repo/yum/2.2/os/RPM-GPG-KEY-EulerOS
```
Comment (jodh-intel) [inline rootfs-builder/euleros/rootfs_lib.sh]: This looks insecure. Does that site support https? If so, please can you change all URs to that. If not, for CentOS we store the key in this repo.

/cc @jcvenegas.

File: rootfs-builder/euleros/rootfs_lib.sh (hunk 1 above)
Comment (grahamwhaley) [reply rootfs-builder/euleros/rootfs_lib.sh]: Looks like it is a modified copy of the original at (this or similar) https://github.com/kata-containers/osbuilder/blob/master/rootfs-builder/centos/rootfs_lib.sh - in which case, yes, probably makes sense to add a Huawei notice in there, just to clarify its history and to make it clear it is not purely Intel code any more.
(IANAL :-))

File: rootfs-builder/euleros/rootfs_lib.sh (hunk 2 above)
Comment (jodh-intel) [reply rootfs-builder/euleros/rootfs_lib.sh]: See #16.

Comment (jcvenegas) [reply rootfs-builder/euleros/rootfs_lib.sh]: @jodh-intel  sounds good, I can update for CentOS

File: rootfs-builder/euleros/rootfs_lib.sh (hunk 1 above)
Comment (WeiZhang555) [reply rootfs-builder/euleros/rootfs_lib.sh]: Yes, please modify it to Huawei.also s/2017/2018/, thanks @liangchenye:smile:

Comment (liangchenye) [reply rootfs-builder/euleros/rootfs_lib.sh]: OK, I'll change it to 2018 Huawei ..

File: rootfs-builder/euleros/rootfs_lib.sh (hunk 2 above)
Comment (liangchenye) [reply rootfs-builder/euleros/rootfs_lib.sh]: Unluckily, we don't have 'https' support yet.  I'll store  the key in github now.

**Comment by liangchenye** [comment]
updated, PTAL @jodh-intel @WeiZhang555 
GPG key is added since we don't support 'https' yet.

**Comment by jodh-intel** [comment]
#18 raised to enable Travis fully for this repo.

lgtm.

@jcvenegas - could you re-review please?

**Comment by jcvenegas** [comment]
@liangchenye  hi this looks good.   just can you add "Signed-off-by" to the commit ?

**Comment by liangchenye** [comment]
'signed-off-by' is added, PTAL @jcvenegas  @jodh-intel

**Comment by jodh-intel** [comment]
Thanks @liangchenye!

lgtm.

**Comment by sameo** [comment]
LGTM

**Comment by jodh-intel** [comment]
Ignoring misleading Travis error and merging...

**Pull request merged**